src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -166,8 +167,9 @@ public void CreateDocument()
         private OpenApiPaths BuildPaths()
         {
             OpenApiPaths pathsCollection = new();
+            RuntimeConfig config = _configProvider.GetConfig();
```

Comment:
should we pass the config here from `CreateDocument()` ? is there a possibility the config that is sourced in `CreateDocument()` is different than the config sourced in `BuildPaths()`?